builder/inference/app.go

```diff
@@ -1,7 +1,8 @@
 package inference
```

Comment:
Should we change the filename to `client.go`?

Reply:
sure